diff --git a/node_modules/expo-router/src/testing-library/mocks.ts b/node_modules/expo-router/src/testing-library/mocks.ts
index 0771fea..3ed6243 100644
--- a/node_modules/expo-router/src/testing-library/mocks.ts
+++ b/node_modules/expo-router/src/testing-library/mocks.ts
@@ -1,4 +1,9 @@
-import "@testing-library/jest-native/extend-expect";
+/*
+ * Optionally enable @testing-library/jest-native/extend-expect. We use this internally for the `toBeOnTheScreen` matcher()
+ */
+try {
+  require('@testing-library/jest-native/extend-expect');
+} catch {}
 
 // include this section and the NativeAnimatedHelper section for mocking react-native-reanimated
 jest.mock("react-native-reanimated", () => {
